test/example_data.dart

```diff
@@ -117,12 +118,57 @@ ZulipStream stream({
     isWebPublic: isWebPublic ?? false,
     historyPublicToSubscribers: historyPublicToSubscribers ?? true,
     messageRetentionDays: messageRetentionDays,
-    streamPostPolicy: streamPostPolicy ?? 1,
-    canRemoveSubscribersGroupId: canRemoveSubscribersGroupId ?? 123,
+    streamPostPolicy: streamPostPolicy ?? StreamPostPolicy.any,
+    canRemoveSubscribersGroup: canRemoveSubscribersGroup ?? 123,
+    streamWeeklyTraffic: streamWeeklyTraffic,
   );
 }
 const _stream = stream;
 
+/// Construct an example subscription from a stream.
+///
+/// We only allow overrides of values specific to the [Subscription], all
+/// other properties are copied from the [ZulipStream] provided.
+Subscription streamSubscription(
+  ZulipStream stream, {
+  List<int>? subscribers,
+  bool? desktopNotifications,
+  bool? emailNotifications,
+  bool? wildcardMentionsNotify,
+  bool? pushNotifications,
+  bool? audibleNotifications,
+  bool? pinToTop,
+  bool? isMuted,
+  String? color,
+  String? emailAddress,
+}) {
+  return Subscription(
+    streamId: stream.streamId,
+    name: stream.name,
+    description: stream.description,
+    renderedDescription: stream.renderedDescription,
+    dateCreated: stream.dateCreated,
+    firstMessageId: stream.firstMessageId,
+    inviteOnly: stream.inviteOnly,
+    isWebPublic: stream.isWebPublic,
+    historyPublicToSubscribers: stream.historyPublicToSubscribers,
+    messageRetentionDays: stream.messageRetentionDays,
+    streamPostPolicy: stream.streamPostPolicy,
+    canRemoveSubscribersGroup: stream.canRemoveSubscribersGroup,
+    streamWeeklyTraffic: stream.streamWeeklyTraffic,
+    subscribers: subscribers ?? [],
+    desktopNotifications: desktopNotifications ?? false,
+    emailNotifications: emailNotifications ?? false,
+    wildcardMentionsNotify: wildcardMentionsNotify ?? false,
+    pushNotifications: pushNotifications ?? false,
+    audibleNotifications: audibleNotifications ?? false,
+    pinToTop: pinToTop ?? false,
+    isMuted: isMuted ?? false,
+    color: color ?? "#FF0000",
+    emailAddress: emailAddress ?? "",
```

Comment:
The empty string doesn't look like a very plausible email address. Here's a better version:
```suggestion
    emailAddress: emailAddress ?? "stream@chat.example",
```

In general the example data in this file should have the right basic shape for real data, at least to the extent that's easy to do.